chart/epinio/templates/helm-controller.yaml

```diff
@@ -0,0 +1,155 @@
+{{- if not (.Capabilities.APIVersions.Has "helm.cattle.io/v1") -}}
```

Comment:
This Deployment doesn't specify a serviceAccountName which means it runs as the default serviceaccount in the kube-system namespace. This serviceaccount may or may not have cluster-admin role bound to it: https://kubernetes.io/docs/reference/access-authn-authz/rbac/#service-account-permissions

If we need to be cluster-admins, we have to be explicit about it. To avoid messing around with the default serviceaccount, we should create a serviceaccount and bind the cluster-admin role to it. Then explicitly use it here. I'll give that a try.

Reply:
Done in c2c4ed634ee9e3d8d33fbf7bd2cfeb89ef67f800